apps/search/test/marionette/lib/search.js

```diff
@@ -124,6 +126,12 @@ Search.prototype = {
     this.client.helper.waitForElement('body');
   },
 
+  searchDataVersion: function() {
+    var file = path.resolve(__dirname +
```

Comment:
Heh I was trying to fix this another way first which is why I ended up parsing json, but thats far simpler, thanks